crypto/params.c

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2019, Oracle and/or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <string.h>
+#include <openssl/params.h>
+
+static const OSSL_PARAM *OSSL_PARAM_locate(const OSSL_PARAM *p, const char *key)
```

Comment:
regarding naming, internal functions are usually not named with upper case prefix...  and actually, if they are functions that you know will always be static, I see no real reason for the prefix, but I'll leave that to your preference.

Reply:
Okay, will change these.